### ISSUE TITLE
Trouble building rustyhex

### ISSUE THREAD

**Issue by lukeschlather** [PistonDevelopers/glfw_window#123]
I tried to build rustyhex, and it has a dependency on obj-rs, which has been renamed to simply obj. After updating the Cargo.toml for that, I still get this error:

$ cargo build
    Updating git repository `https://github.com/csherratt/obj.git`
    Updating git repository `https://github.com/pistondevelopers/event`
Unable to update https://github.com/pistondevelopers/event#58da25cf

Caused by:
  failed to fetch into /home/schlathe/.cargo/git/db/event-64e6f06422add62a

It looks like event has been moved/renamed. What is the migration path here? 

When I run cargo update, it complains it can't find glfw_window 
$ cargo update
    Updating git repository `https://github.com/bjz/cgmath-rs`
    Updating git repository `https://github.com/PistonDevelopers/glfw_window`
    Updating git repository `https://github.com/bjz/gl-rs`
    Updating git repository `https://github.com/dpc/hex2d-rs`
    Updating git repository `https://github.com/pistondevelopers/piston/`
    Updating git repository `https://github.com/gfx-rs/gfx-rs/`
    Updating git repository `https://github.com/csherratt/obj.git`
    Updating git repository `https://github.com/bjz/glfw-rs.git`
no matching package named `glfw_window` found (required by `rustyhex`)
location searched: https://github.com/PistonDevelopers/glfw_window
version required: *

Is Rustyhex not properly declaring its dependency on glfw_window? What is the right way to do this?:

https://github.com/dpc/rustyhex/blob/master/Cargo.toml

**Comment by bvssvni** [comment]
The event crate is moved to the piston repo https://github.com/pistondevelopers/piston

**Comment by lukeschlather** [comment]
Yeah, sorry, I got that. Rustyhex doesn't have a direct dependency on the event crate. What I don't understand is why cargo update fails claiming it can't find glfw_window.
